geometry_msgs/Pose input_pose
---